Dockerfile

```diff
@@ -1,13 +1,14 @@
 FROM golang:1.12.1-alpine3.9 AS build
 
-ENV SAA_RELEASE=master \
+ENV SAA_RELEASE=add-proxy-support \
     SAA_PROJECT=github.com/imduffy15/k8s-gke-service-account-assigner \
-    SAA_GIT_SHA=af04ba0acae0a90faa600390c6de93f521872cf4 \
+    SAA_FORK=github.com/stepanstipl/k8s-gke-service-account-assigner \
```

Comment:
Why did you introduce SAA_FORK instead of updating SAA_PROJECT?

Reply:
This is because how the go build is wired - even though the code comes from my fork, the original location is still hardcoded in many places and therefore I need to clone it to the original place -> both `SAA_FORK` & `SAA_PROJECT` are needed. This is going away anyway, as the PR made it upstream in the meantime :) (https://github.com/imduffy15/k8s-gke-service-account-assigner/pull/8). Sorry for changing this back and forth :).